bin/plot_tree.py

```diff
@@ -104,11 +105,13 @@ def leaf_style(node, seqmeta, tp_colors, highlight_node=None):
             nstyle['fgcolor'] = tp_colors[seqmeta['timepoint']]
         nstyle['size'] = 14
         node.set_style(nstyle)
-    timepoints = filter(lambda x: x, seqmeta.get('cluster_timepoints', seqmeta['timepoints']).split(':'))
-    duplicities = [int(n) for n in seqmeta.get('cluster_timepoint_multiplicities', seqmeta['timepoint_multiplicities']).split(':') if n]
+    timepoints = seqmeta.get('cluster_timepoints', seqmeta['timepoints']).split(':')
+    duplicities = [int(n) for n in seqmeta.get('cluster_timepoint_multiplicities', seqmeta['timepoint_multiplicities']).split(':')]
     multiplicity = int(seqmeta.get('cluster_multiplicity', seqmeta['multiplicity']))
     percents = [d * 100 / multiplicity for d in duplicities]
     colors = [tp_colors[t] for t in timepoints]
+    if len(percents) != len(colors):
```

Comment:
@eharkins All things being equal, I'd put this check up above right away we compute `timepoints` and `duplicities`, and base it on whether the lengths of _those_ two lists are the same.